DEVELOPMENT.md

```diff
@@ -518,43 +515,44 @@ All Github access tokens _must_ have the `public_repo` scope.
     2. Publish the CI artifacts as release candidate binaries.
 
        ```bash
-       ./sripts/release.ts publish-rc-binaries
+       ./scripts/release.ts publish-rc-binaries
        ```
 
     3. Create QA issues for the release that link to the release candidate
        binaries.
 
        ```bash
-       ./sripts/release.ts create-qa-issues
+       ./scripts/release.ts create-qa-issues
        ```
 
     4. Test the release by walking through the QA issues.
     5. (Optional) To fix bugs, create a PR with the fixes based on the release
        candidate branch. Once it has been approved, squash merge it into the
-       release candidate  branch (see [“Merging Pull Requests"][merging-prs])
-       and continue QA.
+       release candidate branch (see [“Merging Pull Requests"][merging-prs]).
+       Then restart the “Test the release” step. (Skip creating a QA
+       issue in 2.2.)
```

Comment:
```suggestion
       issue in 2.3.)
```

Reply:
Fixed in https://github.com/radicle-dev/radicle-upstream/pull/1948/commits/2aa4bd017e979991dbb9825e9bd95e2880dcb1ee